include/ublksrv.h

```diff
@@ -30,6 +30,8 @@ extern "C" {
 #define	DEF_QD		128
 #define	DEF_BUF_SIZE	(512 << 10)
 
+#define JSON_OFFSET   32
```

Comment:
JSON_OFFSET shouldn't be part of API.

Reply:
Done. JSON_OFFSET is not private to lib/ublksrv_json.cpp